src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/WebProfilerExtensionTest.php

```diff
@@ -142,8 +142,6 @@ public function getToolbarConfig()
     }
 
     /**
-     * @group legacy
```

Comment:
Removing this was missed in #33507 where the deprecation of the `intercept_redirects` option was reverted.